packages/graphql-tool-utilities/src/config.ts

```diff
@@ -2,6 +2,7 @@ import {GraphQLConfig, GraphQLProjectConfig} from 'graphql-config';
 import {promisify} from 'util';
 
 import './augmentations';
+import './polyfills';
```

Comment:
this should have been here all along. it was probably getting pulled in by the `ast` module instead which hid the fact that this module was technically broken at runtime (when in isolation).